Management/src/ActuatorApi/Migrations/WeatherDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <auto-generated />
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Steeltoe.Samples.ActuatorApi.Data;
+
+#nullable disable
+
+namespace Steeltoe.Samples.ActuatorApi.Migrations;
+
+[DbContext(typeof(WeatherDbContext))]
+partial class WeatherContextModelSnapshot : ModelSnapshot
```

Comment:
The file was renamed, but not the class itself.

Reply:
Weird, I definitely fixed this yesterday... maybe I accidentally un-did it when removing the copyright headers that got re-added when I did code cleanup